pkg/operator/ceph/object/config.go

```diff
@@ -37,6 +37,10 @@ caps mon = "allow rw"
 caps osd = "allow rwx"
 `
 
+	caBundleVolumeName             = "rook-ceph-rgw-ca-bundle"
+	caBundleDir                    = "/etc/pki/tls/certs"
+	caBundleKeyName                = "cabundle"
+	caBundleFileName               = "ca-bundle.crt"
```

Comment:
@degorenko : I have general concern this will overwrite the default `cabundle.crt` file in RGW pod. But I am not quite sure whether it can have side effects for the general working of the container. It seems to be a bit aggressive for me.
What about adding this under different name int the same location and do `update-ca-trust` with help of init container?

Reply:
actually, you're right, in some cases it may critical. So, i'll change it.

Reply:
@thotz in general, this approach is implemented. So i already tested latest changes on environment. But when init container is completed, updated ca trust chain is not saved for main rgw pod. So, i need to run this command one more time already for rgw pod. Any ideas how we can avoid that?

Reply:
@degorenko Sorry I don't much knowledge about that. In that case, we can avoid init container, just save it as a different file in rgw pod and do `update-ca-trust` in the rgw pod